pkg/ovn/routeadvertisementlist.go

```
@@ -0,0 +1,86 @@
+	if len(options) == 1 {
+		passedOptions = options[0]
+		logMessage += fmt.Sprintf(" with the options %v", passedOptions)
+	}
+
+	glog.V(100).Infof(logMessage)
+
+	routeAdvertisementList := &ovnv1.RouteAdvertisementsList{}
+
+	// Convert metav1.ListOptions to controller-runtime ListOptions
+	listOpts := []runtimeClient.ListOption{}
+	if passedOptions.LabelSelector != "" {
```

golangci-lint (2.2.2)

if statements should only be cuddled with assignments used in the if statement itself

(wsl)